tests/phpunit/unit/tests/AMP/Meta_Sanitizer.php

```diff
@@ -37,9 +37,7 @@ public function set_up(): void {
 
 		Monkey\Functions\stubs(
 			[
-				'wp_parse_url' => static function ( $url, $component = -1 ) {
-					return parse_url( $url, $component ); // phpcs:ignore WordPress.WP.AlternativeFunctions.parse_url_parse_url
-				},
+				'wp_parse_url' => static fn( $url, $component = -1 ) => parse_url( $url, $component ),
```

Comment:
The `// phpcs:ignore WordPress.WP.AlternativeFunctions.parse_url_parse_url` comment probably needs to be kept here